test/models/FileUploadsSpec.scala

```diff
@@ -0,0 +1,79 @@
+/*
+ * Copyright 2021 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package models
+
+import java.time.ZonedDateTime
+
+import models.FileType.SupportingEvidence
+import org.scalatest.{Matchers, WordSpec}
+
+class FileUploadsSpec extends WordSpec with Matchers {
+
+  "FileUploads" should {
+    val MAX = FileUpload.MAX_FILENAME_LENGTH
+
+    "trim the file name" in {
+
+      FileUpload.trimFileName("") shouldBe ""
```

Comment:
there should probably be a test for a filename with more than one dot in the filename  like settings.xml.bak